pkgs/applications/misc/trenchbroom/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, lib, git, wxGTK31, cmake, mesa, mesa_glu, freeimage, freetype, pandoc, libX11 }:
+
+stdenv.mkDerivation rec {
+  pname = "TrenchBroom";
+  version = "2019.6";
+
+  src = fetchgit {
+    url = "https://github.com/kduske/TrenchBroom.git";
+    deepClone = true;
+    rev = "v${version}";
+    sha256 = "1yx09jcz7psg4l9m89amhda30lrclzk74rfp8avbhkz9bypkqn95";
+  };
+
+  buildInputs = [ mesa mesa_glu freeimage freetype pandoc wxGTK31 libX11 ];
+  nativeBuildInputs = [ cmake git ];
+  cmakeFlags = [ "-DwxWidgets_PREFIX=${wxGTK31}" ];
+  hardeningDisable = [ "format" ];
+
+  postInstall = "mkdir $out/share/applications; ln -s ../TrenchBroom/trenchbroom.desktop $out/share/applications";
```

Comment:
- Use multi line syntax for the `postInstall` script
- Consider using `cp` or `install` instead of `ln`